### DISCUSSION TITLE
Skeleton loaders

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#2715] [Annet]
BM har laget en variant for bruk på noen tabeller, men prinsippet er noe vi har lyst til å ha inn i Jøkul. Lager en diskusjon her for å ha et sted å logge litt informasjon.

**Comment by wkillerud** [comment]
Jeg fikk tilsendt koden fra BM (takk @joms og @tomef96!) og tilpasset den bittelitt i #2719 

Forhåndsvisning her: https://jokul.fremtind.no/preview/feat/skeleton-loader/komponenter/loader#skeleton-loader

Kommentarer på koden kan vi gjerne ta i pull requesten, men er det noen innspill på dokumentasjonen eller designet i dark mode (der freestylet jeg litt)?

**Comment by wkillerud** [comment]
Er det noen Figma-skisser et sted vi kan ta i bruk, og få laget disse komponentene i Figma-biblioteket også?

PS: Kanskje skeleton loader-komponenter kan gå som kvasi-wireframes?  👀 🔥 

- #2693

**Comment by wkillerud** [comment]
Har vi noe godt norsk navn for skeleton loaders?

**Reply by joms** [reply]
Random forslag:
- Skjelettlaster
- Skyggelaster
- Innholdslasting
- Ventefyll
- Sidefyll
- Fancyspinner

Konklusjon: nah 🤷‍♂️

**Comment by wkillerud** [comment]
Vi ble enige om i "del 2" på forum i dag å gå for en MVP i Figma. Joachim lager et eksempel og beskrivelser som en del av biblioteket. Tanken er at man som designer bruker det eksempelet som utgangspunkt og lager skjermbildene sine selv. Altså ingen dedikert komponent eller variant i Figma.

Vi diskuterte hvorvidt det ga mening at hver komponent hadde en skeleton-variant, men gjør den jobben hvis det viser seg å bli et behov.
